src/common/schemas/DBRPUpdate.yml

```diff
@@ -1,11 +1,6 @@
   properties:
-    database:
```

Comment:
You can only update the retention policy and "default" designation through the update API, so the "links" and "database" fields are removed here.